src/middlewared/middlewared/plugins/vmware.py

```diff
@@ -634,3 +641,7 @@ def _alert_vmware_login_failed(self, vmsnapobj, e):
 
     def _delete_vmware_login_failed_alert(self, vmsnapobj):
         self.middleware.call_sync("alert.oneshot_delete", "VMWareLoginFailed", vmsnapobj["hostname"])
+
+
+async def setup(middleware):
+    await middleware.call('network.general.register_activity', 'vmware', 'VMware')
```

Comment:
VMWare is probably too broad of a term, should we say VMWare Snapshots or something?